packages/material-ui/src/ClickAwayListener/ClickAwayListener.js

```diff
@@ -10,6 +10,13 @@ function mapEventPropToEvent(eventProp) {
   return eventProp.substring(2).toLowerCase();
 }
 
+function hasClickedScrollbar(event) {
```

Comment:
This works only for scrollbars of the root element (html), right? If so then the function name should reflect that and document its limitation.

Reply:
Correct